docs/user-guide/getting-started.md

```diff
@@ -29,12 +29,12 @@ tar -zxf /path/to/strongbox-distribution*.tar.gz \
     -C /opt/strongbox \ 
     --strip-components=2
 
-# If you just want to start Strongbox without installing the systemd service:
+# If you just want to start Strongbox without installing the system service:
```

Comment:
`systemd` is not a typo. Check [here](https://en.wikipedia.org/wiki/Systemd) for more details.

Reply:
ok